mapviz_plugins/src/point_drawing_plugin.cpp

```diff
@@ -65,6 +65,7 @@ namespace mapviz_plugins
 
   void PointDrawingPlugin::ClearHistory()
   {
+    ROS_INFO("PointDrawingPlugin::ClearHistory(test)");
```

Comment:
```suggestion
    ROS_DEBUG("PointDrawingPlugin::ClearHistory()");
```

Reply:
I'm not sure you meant to resolve this one without changing it...

Reply:
Oops, looks like I had missed one.  My bad, fixed in latest commit.